app/konnect/dev-portal/applications/dynamic-client-registration/azure.md

```diff
@@ -38,31 +38,107 @@ In Azure, create the main application:
 
-1. Sign in to {{site.konnect_short_name}}, then select {% konnect_icon dev-portal %} **Dev Portal** from the menu.
+1. Log in to {{site.konnect_short_name}} and select the **Dev Portal** {% konnect_icon dev-portal %} from the menu.
+
+2. Go to **Application Auth** to find the authentication settings for your API products.
+
+3. Open the **DCR Providers** to review existing DCR Providers.
 
-2. Click **Settings** to open the Dev Portal settings.
+4. Select **New DCR Provider** button to create an Azure configuration. Enter a name for internal reference in {{site.konnect_short_name}}. This name, along with the provider type, will remain internal and not be visible to developers on the Dev Portal.
 
-3. Click the **Application Setup** tab to open the DCR settings for your Dev Portal.
+5. Provide the Issuer URL for your Azure tenant, using the format  `https://sts.windows.net/YOUR_TENANT_ID`.
 
-4. Select **Azure** as the external identity provider..
+6. For **Provider Type**, select AzureAD.
 
-5. Enter the Issuer for your Azure tenant, it will look something like `https://sts.windows.net/YOUR_TENANT_ID`.
+7. Enter your Application (Client) ID from Azure into the **Initial Client ID** field and the Azure admin application's client secret into the **Intial Client Secret** field. The Initial Client Secret will be stored in isolated, encrypted storage and will not be readable through any Konnect API.
 
-6. Enter `appid` in the **Consumer claims** field.
+8. After saving, your new DCR Provider will appear in the list.
 
-7. Select the auth method you want to enable.
+9. Click the **Auth Strategy** tab, then select **New Auth Strategy** to create an auth strategy that uses the DCR Provider you just added.
 
-8. Enter your Application (client) ID from Azure in the **Initial Client ID** field.
+10. Name it for internal reference in {{site.konnect_short_name}} and provide a display name for visibility in the Portal. Select DCR as the **Auth Type** and choose your newly created DCR provider from the dropdown. The  **Issuer URL** entered earlier will be will be prepopulated.
 
-9. Enter the Client secret from the admin application created in Azure into the **Initial Client Secret** field..
+11. In the **Credential Claims** field, enter `appid`.
 
-10. Click **Save**.
+12. Select the relevant **Auth Methods** you need (`client_credentials`, bearer, session), and **save**. 
 
-   If you previously configured any DCR settings, this will
-   overwrite them.
+{% endnavtab %}
+{% navtab API %}
+After configuring Azure, you can integrate it with the Dev Portal for dynamic client registration (DCR). This process involves two steps: creating the DCR provider and establishing the authentication strategy. DCR providers are designed to be reusable configurations. This means once you've configured the Azure DCR provider, it can be utilized across multiple authentication strategies without needing to be set up again.
```

Comment:
```suggestion
After configuring Azure, you can integrate it with the Dev Portal for dynamic client registration (DCR). This process involves two steps: creating the DCR provider and establishing the authentication strategy. DCR providers are designed to be reusable configurations. This means once you've configured the Azure DCR provider, it can be used across multiple authentication strategies without needing to be set up again.
```